rllib/algorithms/algorithm_config.py

```diff
@@ -330,8 +329,8 @@ def __init__(self, algo_class=None):
 
         # `self.training()`
         self.gamma = 0.99
+        self.optimizer_type = "adam"
         self.lr = 0.001
-        self.lr_schedule = None
```

Comment:
`lr_schedule` is no longer supported on the new stack, but still on those old-stack algos that had this as an algo-specific setting, e.g. PPO (on old stack).
For the new stack, users should simply use `lr: [ls-schedule setup]`.